app/views/sessions/welcome.html.erb

```diff
@@ -0,0 +1,70 @@
+
+<% if logged_in? %>
+  <div class="jumbotron text-center" style="margin-bottom:0">
+    <h1>Welcome, <%= current_admin.user %>!</h1> 
+  </div>
+
+  <nav class="navbar navbar-expand-sm bg-dark navbar-dark">
+    <a class="navbar-brand" href="#">Main</a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#collapsibleNavbar">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="collapsibleNavbar">
+     <ul class="navbar-nav">
+        <li class="nav-item">
+          <a class="nav-link" href="employees#show">Employees</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="branches#show">Branches</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="reports#show">Reports</a>
+        </li> 
+        <li class="nav-item">
+          <a class="nav-link", <%= link_to "Log out",:controller=> "sessions", :action=> "destroy"%></a> 
+          
+        </li>    
+      </ul>
+    </div>  
+  </nav>
+<%end%>
+
+
+<!-- Full Page Intro -->
+<div class="view img-fluid welcome_img" style="background-image: url('https://bit.ly/34XuUBR'); background-repeat: no-repeat; background-size: cover; background-position: center center;">
```

Comment:
You can use the bootstrap class `h-100` rather than creating `welcome_img` since they do basically the same 😄